include/config.h

```diff
@@ -342,7 +350,7 @@
  *      died due to program or system crashes to be resumed from the point
  *      of the last level change, after running a utility program.
  */
-#define INSURANCE /* allow crashed game recovery */
+/* #define INSURANCE allow crashed game recovery */
```

Comment:
meant to stay commented?

Reply:
Yes, this is related to file locks which we don't want for several reasons.